open-metadata-implementation/repository-services/repository-services-implementation/src/main/java/org/odpi/openmetadata/repositoryservices/archivemanager/opentypes/OpenMetadataTypesArchive.java

```diff
@@ -10257,7 +10275,7 @@ private EntityDef getPortAliasEntity()
          */
         final String guid            = "BgehpbcK-M5NI-KQ4R-ElaX-cmzumvM0AW8k";
```

Comment:
Wouldn't we normally use a hex guid (I'll not note any others). technically works fine, but it seems somewhat non standard?